libraries/AP_Scripting/applets/Aerobatics/FixedWing/plane_aerobatics.lua

```diff
@@ -1812,6 +2190,168 @@ function calculate_rudder_offset(ahrs_quat, ahrs_gyro, airspeed_constrained)
    return rudder_ofs
 end
 
+--[[
+   convert a table of bytes to a lua string
+--]]
+function bytes_to_string(bytes)
```

Comment:
I think this could just be `string.unpack('z')` to get a zero terminated string. Or maybe `string.unpack('c10')` to get a string of length 10.

Reply:
the problem is it the mavlink parser produces a table of integers so string.unpack doesn't work